pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -164,9 +164,9 @@ func (m *kubeGenericRuntimeManager) generateLinuxContainerResources(pod *v1.Pod,
 // Swap is only configured if a swap cgroup controller is available and the NodeSwap feature gate is enabled.
 func (m *kubeGenericRuntimeManager) configureContainerSwapResources(lcr *runtimeapi.LinuxContainerResources, pod *v1.Pod, container *v1.Container) {
 	if !swapControllerAvailable() {
-		klog.InfoS("No swap cgroup controller present", "swapBehavior", m.memorySwapBehavior, "pod", klog.KObj(pod), "containerName", container.Name)
```

Comment:
just wondering, would it make sense to keep the log but bump the V level to 5 or more?

Reply:
On the one hand I think it does, since it would be much more visible that every container cannot use swap.
On the other hand I think it doesn't, as this log is not providing any new information, just repeats itself for every container.

I guess the question is how we define high visibility mode (i.e. V>=5). Do we expect that logs would be more visible, or do we expect more information that's rarely necessary?

Reply:
But I guess that high visibility is set for debugging, and making this log appear on every container's logs might help while not massively spamming logs. So I tend to think raising V is the right way to go.

@liggitt WDYT?

Reply:
Thanks for elaborating! I don't have strong opinions, so by all means I'm fine with removing the log. I'm seeing what a believe is a trend in reporting mismatched configuration betweek kube/kubelet and the system, for example runtime lacking features (runtime too old/disabled), or system missing features like this case. We do log them, and this is fine, but we as SIG should perhaps think about a mechanism to make these conditions more visible. The reason why I'm mentioning this thought is that repeating the log makes at least the condition more visible, and doing at a high log level keeps the spam at bay. But this is a larger conversation and I don't want to drag the conversation too long, so from my PoV the question I had is answered and we can carry along.

Reply:
Do we still log anything in that regard?

Reply:
@fabiand Yes indeed. The log here would still fire, but will do so only once:
https://github.com/kubernetes/kubernetes/blob/d9c54f69d4bb7ae1bb655e1a2a50297d615025b5/pkg/kubelet/kuberuntime/kuberuntime_container_linux.go#L343-L366